plugins/outputs/sensu/README.md

```diff
@@ -0,0 +1,69 @@
+# HTTP Output Plugin
+
+This plugin is based off of the HTTP output plugin to send metrics
+to the Sensu Events API.
+
+### Configuration:
+
+```toml
+  ## Configure check configurations
+  [outputs.sensu-go.check]
+    name = "telegraf"
```

Comment:
Can you please detail a bit more on what it does? What is the `name` parameter about?